tests/api_app/test_api.py

```diff
@@ -23,6 +24,7 @@ def setUpClass(cls):
         cls.superuser = User.objects.create_superuser(
             username="test", email="test@intelowl.com", password="test"
         )
+        call_command("migrate_secrets")
 
     def setUp(self):
         self.client = APIClient()
```

Comment:
could you add tests for the new endpoints you created here please?

Reply:
I've made them in `tests/api_app/test_org_plugin_state.py`. Should I copy them here?

Reply:
ah no no it's better like you did. I did not notice.